docs/runbooks/extending_workspace_time/extending_workspace.md

```diff
@@ -35,3 +35,20 @@ Following the instructions for editing the workspace Time To Live will put this
 
 This will not work if there is no record in place.
 At this point it's recommended to create the PR environment through the normal CI Process of bumping the PR via a code change or merge.
+
+## removing workspace
+When UAT is completed, it might be a good idea to remove the environment so that we're being more efficient with the cost.
```

Comment:
pretty much yep:
```suggestion
When manual testing is completed on UAT, it might be a good idea to remove the environment so that we're being more efficient with the cost.
```